commit 81bbcc468880909d3b71ae47b0da1619c0aabfe5
Author: Sunil Thaha <sunil.thaha@nokia.com>
Date:   Mon Mar 7 15:28:16 2011 +1000

    compile on MeeGo
    
    Ugh. Harmattan and MeeGo have different versions of an external library.
    I can't see a nice way to work around this so for now, just use an ifdef.
    This will most likely break the MeeGo build again once it gets the updated
    package from Harmattan.
    
    Conflicts:

diff --git a/plugins/organizer/mkcal/mkcalengine.cpp b/plugins/organizer/mkcal/mkcalengine.cpp
index 022496a..abbc725 100644
--- a/plugins/organizer/mkcal/mkcalengine.cpp
+++ b/plugins/organizer/mkcal/mkcalengine.cpp
@@ -232,7 +232,11 @@ QList<QOrganizerItem> MKCalEngine::internalItemOccurrences(const QOrganizerItem&
             maxCount);
     KCalCore::Recurrence *recurrence = generatorIncidence->recurrence();
     foreach (const mKCal::ExtendedCalendar::ExpandedIncidence& expandedIncidence, incidenceList) {
+#ifdef Q_WS_MEEGO
         QDateTime incidenceDateTime = expandedIncidence.first;
+#else
+        QDateTime incidenceDateTime = expandedIncidence.first.dtStart;
+#endif
         KCalCore::Incidence::Ptr incidence = expandedIncidence.second;
         QOrganizerItem instance;
         //unfortunately sometimes expandRecurrences will include also those occurrences which are in exDates
